docs/rules.md

```diff
@@ -980,6 +974,12 @@ For any PUT, response code should be 201 if resource was newly created and 200 i
 
 Please refer to [put-resonse-schema-description.md](./put-resonse-schema-description.md) for details.
 
+### PutResponseCodes
+
+Synchronous and long-running PUT operations must have responses with 200, 201 and default return codes. They also must not have other response codes.
```

Comment:
Bold or  underline the `not` in "must not have other ..."

Reply:
same for this as well